src/main/kotlin/org/rust/ide/inspections/lints/RsNamingInspection.kt

```diff
@@ -36,10 +36,17 @@ abstract class RsNamingInspection(
         holder.registerLintProblem(
             id,
             "$elementType `$name` should have $styleName case name such as `$suggestedName`",
-            *fixes)
+            *fixes
+        )
     }
 
     abstract fun checkName(name: String): Pair<Boolean, String?>
+
+    companion object {
+        internal val OK = Pair(true, null)
+
+        internal val Char.hasCase: Boolean get() = isLowerCase() || isUpperCase()
```

Comment:
Looks like we actually need this property only in `RsCamelCaseNamingInspection`, so let's move it into `RsCamelCaseNamingInspection` and make it private